gadget/install/install.go

```diff
@@ -176,12 +176,21 @@ func Run(gadgetRoot, device string, options Options) error {
 		loadChain = append(loadChain, options.KernelPath)
 	}
 
-	// TODO:UC20: get cmdline definition from bootloaders
+	// Get the expected kernel command line for the system that is currently being installed
+	cmdline, err := boot.ComposeCandidateCommandLine(options.Model)
```

Comment:
It should be possible to test is by at least mocking the static command line snippets, eg.

```
import "github.com/snapcore/snapd/bootloader/assets"
...
assets.MockSnippetsForEdition("grub.cfg", []assets.ForEditions{
    Edition: 1, Snippet: []byte("grub.cfg static part"),
})
assets.MockSnippetsForEdition("grub-recovery.cfg", []assets.ForEditions{
    Edition: 1, Snippet: []byte("grub-recovery.cfg static part"),
})
```

but I don't see any mocking of secboot.SealKey(). Perhaps a followup?

Reply:
Let's add more extensive testing it in a followup,